backend/visualization/wordcloud.py

```diff
@@ -8,7 +8,7 @@
 def field_stopwords(corpus_name, field_name):
     corpus = Corpus.objects.get(name=corpus_name)
     field = corpus.configuration.fields.get(name=field_name)
-    if field.language and field.language is not 'dynamic':
+    if field.language and field.language != 'dynamic':
```

Comment:
Linters will actually replace `!=` by `is not`. Does `is not` cause a bug here?

Reply:
I wasn't really aware of this either, but yes, `is not` checks for identity rather than equality, which caused the issue.

From https://stackoverflow.com/questions/2209755/python-operation-vs-is-not

>  When comparing to `None`, always use is not. None is a singleton in Python - there is only ever one instance of it in memory.
> 
> By comparing identity, this can be performed very quickly. Python checks whether the object you're referring to has the same memory address as the global None object - a very, very fast comparison of two numbers.
> 
> By comparing equality, Python has to look up whether your object has an __eq__ method. If it does not, it examines each superclass looking for an __eq__ method. If it finds one, Python calls it. This is especially bad if the __eq__ method is slow and doesn't immediately return when it notices that the other object is None.
> 
> Did you not implement __eq__? Then Python will probably find the __eq__ method on object and use that instead - which just checks for object identity anyway.
> 
> When comparing most other things in Python, you will be using **!=.**